internal/db/update/update.go

```diff
@@ -61,6 +62,32 @@ func (m *SchemaUpdateManager) AppendSchema(extensions []schema.Update) {
 	m.updates[updateExternal] = extensions
 }
 
+func updateFromV2(ctx context.Context, tx *sql.Tx) error {
+	stmt := `
+CREATE TABLE internal_cluster_members_new (
+    id                      INTEGER   PRIMARY  KEY    AUTOINCREMENT  NOT  NULL,
+    name                    TEXT      NOT      NULL,
+    address                 TEXT      NOT      NULL,
+    certificate             TEXT      NOT      NULL,
+    schema_internal         INTEGER   NOT      NULL,
+  	schema_external         INTEGER   NOT      NULL,
+    heartbeat               DATETIME  NOT      NULL,
+    role                    TEXT      NOT      NULL,
+    api_extensions          TEXT      NOT      NULL DEFAULT '{}',
```

Comment:
Since extensions are a `[]string`, shouldn't the default value be `[]`?

Reply:
yes it should! I'm testing this with MicroOVN right now and I'm running into an unmarshalling issue when upgrading a legacy cluster. Thanks!